_partials/_caggs-intro.md

```diff
@@ -8,12 +8,19 @@ temperature readings taken every second, you can find the average temperature
 for each hour. Every time you run this query, the database needs to scan the
 entire table and recalculate the average every time.
 
-Continuous aggregate views are refreshed automatically in the background as new
-data is added, or old data is modified. Timescale tracks these changes to the
-dataset, and automatically updates the view in the background. This does not add
-any maintenance burden to your database, and does not slow down `INSERT`
-operations.
+Continuous aggregates are a kind of hypertable that is refreshed automatically
+in the background as new data is added, or old data is modified. Changes to your
```

Comment:
@Loquacity some thoughts for you to decide where to add: 

I would add here (or wherever you think that fits best) a mention that continuous aggregates are like materialized views, with the main difference that they are **_continuously_** and **_incrementally_** updated, hence the "continuous" part of the name. The fact that they are _incrementally_ updated also makes them much less expensive to keep up to date compared to regular materialized views that are all or nothing and have to be refreshed and recalculated from scratch every time they have to be updated (which can be super expensive over months or years of data). 

The mention to materialized views may be important both for SEO (advanced PostgreSQL users look for materialized views) and for completeness (as we mention ` materialized view` at the last paragraph of this page without having mentioned the fact that continuous aggregates are materialized views)